package/pom.xml

```diff
@@ -53,22 +53,64 @@
   <build>
     <plugins>
       <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
         <configuration>
-          <finalName>
-            ${jar.assembly.name.prefix}-spark${sparkbundle.version}_${scala.binary.version}-${project.version}
-          </finalName>
+          <finalName>gluten-package-${project.version}</finalName>
         </configuration>
         <executions>
           <execution>
-            <id>make-assembly</id>
             <phase>package</phase>
             <goals>
-              <goal>single</goal>
+              <goal>shade</goal>
             </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.protobuf</pattern>
+                  <shadedPattern>io-glutenproject.shaded.com.google.protobuf</shadedPattern>
```

Comment:
It's not that usual to use hyphen in package naming. Let's use `io.glutenproject` directly.